src/pages/pool/_id.vue

```diff
@@ -308,9 +310,14 @@ export default defineComponent({
     const missingPrices = computed(() => {
       if (pool.value) {
         const tokensWithPrice = Object.keys(prices.value);
-        return !pool.value.tokenAddresses.every(token =>
-          tokensWithPrice.includes(token)
-        );
+
+        const tokens =
+          isStablePhantomPool.value &&
+          pool.value.linearPoolTokensAddresses != null
+            ? pool.value.linearPoolTokensAddresses
+            : pool.value.tokenAddresses;
```

Comment:
This is a little hard to parse, maybe move the conditional into an understandable const like:
```suggestion
        const hasLinearPoolTokens =  isStablePhantomPool.value && pool.value.linearPoolTokensAddresses != null
        const tokens = hasLinearPoolTokens
            ? pool.value.linearPoolTokensAddresses
            : pool.value.tokenAddresses;
```